src/Handlers.cpp.Rt

```diff
@@ -897,49 +897,105 @@ class cbFailcheck: public Callback {
 		reg.dx = solver->region.dx;
 		reg.dy = solver->region.dy;
 		reg.dz = solver->region.dz;
-               	pugi::xml_attribute attr = node.attribute("dx");
-                if (attr) {
-                       	reg.dx = solver->units.alt(attr.value());
-                }
-                attr = node.attribute("dy");
-                if (attr) {
-                        reg.dy = solver->units.alt(attr.value());
-                }
-                attr = node.attribute("dz");
-                if (attr) {
-                       	reg.dz = solver->units.alt(attr.value());
-                }
+	
+
+
+        pugi::xml_attribute attr = node.attribute("dx");
+        if (attr) {
+            reg.dx = solver->units.alt(attr.value());
+        }
+        attr = node.attribute("dy");
+        if (attr) {
+            reg.dy = solver->units.alt(attr.value());
+        }
+        attr = node.attribute("dz");
+        if (attr) {
+            reg.dz = solver->units.alt(attr.value());
+        }
+
+
+        attr = node.attribute("nx");
+        if (attr) {
+            reg.nx = solver->units.alt(attr.value());
+        }
+        attr = node.attribute("ny");
+        if (attr) {
+            reg.ny = solver->units.alt(attr.value());
+        }
+        attr = node.attribute("nz");
+        if (attr) {
+            reg.nz = solver->units.alt(attr.value());
+        }
+
 		rkept = 1;
 		return 0;
 	}
 	int DoIt() {
 		Callback::DoIt();
-        	int ret = 0;
-		int cond,fin;
+       	int ret = 0;
+		int cond,fin,go;
 		fin = false;
 		cond = false;
+        go = false;
+
+        pugi::xml_attribute comp = node.attribute("check_quantities");
```

Comment:
if we want to specify what kind of quantities, we want to check, it is better to use `node.attribute("what")` . "what" is already used in many handlers, lets be consistent